tests/test_pytest_rerunfailures.py

```diff
@@ -472,7 +472,7 @@ def pytest_runtest_teardown(item):
             assert item.execution_count == 3"""
     )
     result = testdir.runpytest("--reruns", "2")
-    assert_outcomes(result, passed=3, rerun=2)
+    assert_outcomes(result, passed=1, rerun=2)
```

Comment:
The test name says that `execution_count` is exposed to the test runner. So it should be `3` passed tests, as this is set during tear down.

I do not know why this behaviour is here and why it breaks now.